src/_includes/graphql/gift-registry.md

```diff
@@ -0,0 +1,73 @@
+The `giftRegistry` object returns the following attributes:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`created_on` | String! | The date on which the gift registry was created. Only the registry owner can access this attribute
+`dynamic_attributes` | `[GiftRegistryDynamicAttribute]` | An array of attributes that define elements of the gift registry. Each attribute is specified as a code-value pair
+`event_name` | String! | The name the customer assigned to the event
+`id` | ID! | The ID assigned to the gift registry
+`items` | [[GiftRegistryItemInterface](#GiftRegistryItemInterface)] | An array of products added to the gift registry
+`message` | String! | The message text the customer entered to describe the event
+`owner_name` | String! | The customer who created the gift registry
+`privacy_settings` | `GiftRegistryPrivacySettings!` | An enum that states whether the gift registry is PRIVATE or PUBLIC. Only the registry owner can access this attribute
+`registrants` | [[GiftRegistryRegistrant](#GiftRegistryRegistrant)]| Contains details about each registrant for the event
+`shipping_address` | CustomerAddress | Contains the customer's shipping address. Only the registry owner can access this attribute
```

Comment:
Hi @keharper. Could you clarify, please, is there a technical reason of wrapping only some of the types into `backquotes`? 

Thank you.

Reply:
I.e. `GiftRegistryPrivacySettings!`vs CustomerAddress

Reply:
They were marked that way in one of the original PRs, and I didn't notice it. I'll correct them.